extended.md

```diff
@@ -722,15 +645,17 @@ Jede Rolle ist wichtig, egal ob Bürger oder Mafia Goon, also verschwendet sie n
 
       *Eingereicht von Blubb*
 
-  - ### **Bürgermeister**
-      {: #buergermeister}
-
-      **Alias**: Doublevoter
+  - ### **Trickster**
+      {: #trickster}
 
-      **Fähigkeit**: Der Bürgermeister funktioniert wie eine zweite Rolle. Sie wird von dem Spielleiter zufällig verteilt am Anfang des Spiels. Der Spieler der die Bürgermeister Rolle zusätzlich bekommen hat eine doppelte Stimme in der Wahl des Lynchens.
+      **Fähigkeit**: Jede Nacht wählt er zwei Spieler aus, diesen zwei Spielern passieren zufällige Aktionen. Dabei kann der Spielerleiter auswählen was diese zufälligen Aktionen sind. Der Trickster gewinnt, wenn alles Spieler tot sind. 
 
-      **Schwierigkeitsgrad**: Mittel - Anspruchsvoll
+      **Tipps für den *Spielleiter***:
 
-      **Tipp für den *Spielleiter***: Die Identität des Bürgermeisters ist immer bekannt.
+	  * Du solltest auch schauen das auch zwei Spieler mit einander verlinkt werden können, mittels der Fähigkeit des Amors.
+	  * Sollte ein Spieler zum Kultmitglied gemacht wird der Trickster als Kultmeister gezählt. 
+	  
+   - ### **Vampirmeister**
+      {: #vampirmeister}
 
-      *Eingereicht von Bujuhu*
+      **Fähigkeit**: Der Vampirmeister funktioniert ähnlich wie der Kultmeister, er sucht jede Nacht einen Spieler aus der zum Vampir wird und auch dem Chat vom Vampirmeister beitritt. Sollte er sterben sterben auch seine Vampiranhänger. Sollten folgene Rollen den Vampirmeister oder ein Vampiranhänger in der Nacht wählen, stirbt dieser sofort: Hynotiseur, Detektiv, Cop, Arzt, Maler und Amor.
```

Comment:
"Der Vampirmeister funktioniert ähnlich wie der Kultmeister: Er sucht jede Nacht einen Spieler aus, welcher zum Vampir wird und auch dem Chat des Vampirmeisters beitritt. Sollte der Vampirmeister sterben, sterben alle seine Vampiranhänger mit ihm. Sollten folgende Rollen den Vampirmeister oder einen Vampiranhänger in der Nacht als Ziel ihrer Fähigkeit wählen, stirbt dieser sofort: Hypnotiseur, Detektiv, Cop, Arzt, Maler und Amor."